### PR TITLE
Needles for new desktop selection workflow

### PR DESCRIPTION
**Pull request by nilxam**

### CODE REVIEW

File: gnome-selected-20170316.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    {
+      "width": 162,
+      "type": "match",
+      "xpos": 358,
+      "ypos": 373,
+      "height": 12
+    }
+  ],
+  "properties": []
+}
```
Comment (okurz) [inline gnome-selected-20170316.json]: technically the needle is fine but the design of the installer page is not very convincing. This was also mentioned to yast team in mailing list threads, irc channel, pull requests. I don't know how to continue here but I am objecting the acceptance of the state as is.

@nilxam not your fault

File: installer_desktopselection-has-server-selection-text-20170317.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    {
+      "type": "match",
+      "width": 167,
+      "xpos": 52,
+      "ypos": 367,
+      "height": 16
+    }
+  ],
+  "tags": [
+    "has-server-selection"
```
Comment (okurz) [inline installer_desktopselection-has-server-selection-text-20170317.json]: I am not quite sure if maybe at least this screen is already influenced by https://github.com/yast/yast-installation/pull/537

Comment (nilxam) [inline installer_desktopselection-has-server-selection-text-20170317.json]: I'll try iso with 3.2.30 built.

File: gnome-selected-20170316.json (hunk 1 above)
Comment (nilxam) [inline gnome-selected-20170316.json]: well, release team decide to take it up first if the function is worked. all discussion about UI improvement can be continue, but that is NOT the reason blocked this, and not a reason rejecting it from openqa POV.

and superseding request is catching us.

Comment (lnussel) [inline gnome-selected-20170316.json]: first step is to get the functionality in. design is second.

**Pull request merged**